.circleci/config.yml

```diff
@@ -28,7 +40,19 @@ jobs:
       - aws-cli-configure
       - sam-cli-install
       - checkout
-      - build-repo-s3-upload
+      - build-repo
+
+  build-deploy-copy:
+    executor: ubuntu-2004
+    steps:
+      - brew-install
+      - aws-cli-configure
+      - aws-cli-configure-prod
+      - sam-cli-install
+      - checkout
+      - build-repo
+      - copy-to-s3
+      - copy-to-s3-prod
```

Comment:
Can we make sure the prod part only happens on the `master` branch?

Reply:
The build-deploy workflow doesn't copy to S3 now for either prod or sandbox.  The build-deploy-copy workflow has a filter to only run when the branch is set to master.

Reply:
Ok great, thanks for confirming